tools/etl/tg-s3-parquet-load/compile_and_run.sh

```diff
@@ -13,7 +13,7 @@ target_path="$cwd/uber_data"
 times=1
 log_path=""
 if [[ $# < 3 ]]; then
-  echo "Usage: ./compile_and_run.sh  master_url target_host target_path [times] [log_path]"
+  echo "Usage: ./compile_and_run.sh  master_url target_host target_path [times] [log_path] [properties_file]"
```

Comment:
after specify properties_file, where is it being read? and how it is integrated?

Reply:
It is being read by the same script compile_run.sh (line 33-36). In the same script the default file will be used (line 8) if this optional property is not provided. The integration is already there in the same script and the property file is being passed to the generateDataset.java file (line 45-48).